### PR TITLE
Get 35 landing page

### PR DESCRIPTION
**Pull request by tavy88**
### Context

Set up the home page and link it to Jobs Search.

### Screenshots

Screen Shot 2019-06-19 at 09 55 30

Screen Shot 2019-06-19 at 10 08 35

### Ticket

JIRA - https://dfedigital.atlassian.net/browse/GET-35

### CODE REVIEW

**Comment by cwrw** [comment]
I've tried looking around for best practices, and there seems to be none. I've been thinking about this, and maybe it might be better to have it under the pages controller as we might have further static pages, which could be served up under the "pages" group where they aren't the "home" page.

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<div class="govuk-grid-row govuk-!-padding-top-4">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= t('home_page.title') %></h1>
+    <p class="govuk-body">Use this service to find out what you need to do to retrain for another job.</p>
```
[the remark this answers is not in the document]
Comment (tavy88) [reply app/views/home/index.html.erb]: Don't think this content will be shared by any other page so there will always be the same place to change it. Spoke earlier with @smoothcontract as well. Also, the markup can change at any point causing a lot of pain afterwards in the locale file keys.

[the remark this answers is not in the document]
Comment (tavy88) [reply app/views/home/index.html.erb]: I hope that in future we can move towards a CMS.

[the remark this answers is not in the document]
Comment (smoothcontract) [reply app/views/home/index.html.erb]: We only need to support English and England so don’t need to go overboard with localising everything. That said it’s nice to keep controllers, helpers, decorators etc. clean without long hardcoded text strings. So I think it’s appropriate for this project to use i18n in moderation but no need to rigidly enforce use of it for all content.

**Pull request merged**